src/background.js

```diff
@@ -104,9 +104,15 @@ class StudyLifeCycleHandler {
    */
   async handleStudyEnding(ending) {
     console.log(`Study wants to end:`, ending);
-    for (const url of ending.urls) {
+    const user = await browser.fxa.getSignedInUser();
+    const studyInfo = await browser.study.getStudyInfo();
```

Comment:
Do you need the `await` here? The [docs](https://github.com/mozilla/shield-studies-addon-utils/blob/master/webExtensionApis/study/api.md#browserstudygetstudyinfo-) don't seem to mention `getStudyInfo` returning a promise.